notifier/notifier.go

```diff
@@ -231,6 +236,18 @@ func (n *Notifier) Send(alerts ...*model.Alert) {
 	n.setMore()
 }
 
+func (n *Notifier) relabelAlerts(alerts []*model.Alert) []*model.Alert {
+	var relabeledAlerts []*model.Alert
+	for _, alert := range alerts {
+		labels, _ := retrieval.Relabel(alert.Labels, n.opts.RelabelConfigs...)
```

Comment:
You wanted to remove the error from that function signature, right.
Let's move the relabel functionality into its own package while we are at it. `retrieval` has no logical relation with notifications.